index.html

```diff
@@ -520,7 +521,7 @@ <h3>Integrity Validation</h3>
         <p>
           An example of such usage is provided below:
         </p>
-        <pre class="example nohighlight" title="Example Verifiable Credential JsonSchema2023  with Integrity Information">
```

Comment:
This breaks the highlighting of the example. Was that intentional?

Reply:
the rest of the sections do not have this property, according to [respec docs](https://respec.org/docs/#nohighlight) this property _removes_ highlighting...have you seen different behavior?

Reply:
The behavior I'm describing is the ability to highlight text within a `<pre>` block. 

You can see below how a `<pre class="example nohighlight">` block is currently rendered:
![image](https://github.com/w3c/vc-json-schema/assets/1139998/94a3ff1e-41f6-4028-b4a7-9c7e5581255b)

When you remove the `nohighlight` (i.e. `<pre class="example">`) then it's rendered as below, regardless of having spans within the `<pre>` block:
![image](https://github.com/w3c/vc-json-schema/assets/1139998/eadfa104-f9a6-4407-9992-a317f579885b)

Reply:
Here's how the same example is rendered after the changes you're proposing. Compare this against the first image I pasted. 

![image](https://github.com/w3c/vc-json-schema/assets/1139998/c0bfd2a4-3bfd-4716-a036-6d84eea5f794)

Reply:
ahhh gotcha, reverted